tests/server.test.js

```diff
@@ -144,6 +144,17 @@ test('Check /friends', t => {
   });
 });
 
+test('Check /school', t => {
+  const options = {
+    method: 'GET',
+    url: '/school'
+  };
+  server.inject(options, response => {
+    t.equal(response.statusCode, 200, 'You received a 200 status code');
```

Comment:
Ok I think you can start making your tests a bit more sophisticated.

Not exactly related here but can we test the hsb partial buttons? I should have picked up on this previously, sorry to be retrospectively asking you to test stuff.